web/static/apps/wos/wos.js

```diff
@@ -0,0 +1,37 @@
+const REFRESH_INTERVAL = 60000; //ms
+
+async function display_wall() {
+  let response = await fetch("/api/wos/users");
+  let content = document.getElementById("content");
+  try {
+    let users_data = await response.json();
+    if (users_data.length == 0) throw "No user in debt";
+    let table =
+      `<table>
+        <tr>
+          <th>Name</th>
+          <th>Debt($)</th>
+        </tr>` +
+      users_data
+        .map(
+          (users) =>
+            `<tr>
+                <td>${users.username}</td>
+                <td class="balance">${users.balance}</td>
```

Comment:
Balance is given in cents, so we need to do some math on it to display it. For now you can copy the definition of `dollars` that we use in `pos` and I'll figure out how to deduplicate the definition later.


```js
export function dollars(cents) {
  let d = Math.abs(Math.trunc(cents / 100));
  let c = Math.abs(cents) % 100;
  return `${cents < 0 ? "-" : ""}${d}.${c < 10 ? "0" + c : c}`;
}
```